oeem_etl/storage.py

```diff
@@ -49,11 +49,11 @@ def _make_target_classes(self):
 
         class TargetWithClient(target):
             def __init__(self, path):
-                super().__init__(path, client=client, format=MixedUnicodeBytesFormat())
+                super(TargetWithClient, self).__init__(path, format=MixedUnicodeBytesFormat())
```

Comment:
GCS and S3 will both need the client keyword arg, even though the LocalTarget doesn't need it, so we need to manage those cases as well here and when making the flag target below.